File: opcua-webservice-spring/core/src/main/java/org/opcfoundation/webserver/addressspace/nodes/builtin/UaObjectTypes.java
package org.opcfoundation.webserver.addressspace.nodes.builtin;

import org.eclipse.milo.opcua.stack.core.NodeIds;
import org.eclipse.milo.opcua.stack.core.types.builtin.LocalizedText;
import org.eclipse.milo.opcua.stack.core.types.builtin.NodeId;
import org.opcfoundation.webserver.addressspace.nodes.UaObjectType;

public class UaObjectTypes {
    public final static UaObjectType BaseObjectType = new UaObjectType(NodeIds.BaseObjectType,"BaseObjectType", new LocalizedText("BaseObjectType"), false);
    public final static UaObjectType FolderType = new UaObjectType(NodeIds.FolderType,"FolderType", new LocalizedText("FolderType"), false);
    public final static UaObjectType ServerType = new UaObjectType(NodeIds.ServerType,"ServerType", new LocalizedText("ServerType"), false);

    public final static UaObjectType ModellingRuleType = new UaObjectType(NodeIds.ModellingRuleType,"ModellingRuleType", new LocalizedText("ModellingRuleType"), false);

    public final static UaObjectType BaseEventType = new UaObjectType(NodeIds.BaseEventType,"BaseEventType", new LocalizedText("BaseEventType"), false);

    public final static UaObjectType DigitalTwinType = new UaObjectType(new NodeId(1,10),"DigitalTwinType", new LocalizedText("DigitalTwinType"), true);
    public final static UaObjectType DigitalTwinRepositoryType = new UaObjectType(new NodeId(1,11),"DigitalTwinRepositoryType", new LocalizedText("DigitalTwinRepositoryType"), true);
    public final static UaObjectType SubmodelType = new UaObjectType(new NodeId(1,12),"SubmodelType", new LocalizedText("SubmodelType"), true);
    public final static UaObjectType ElementType = new UaObjectType(new NodeId(1,13),"ElementType", new LocalizedText("ElementType"), true);
    public final static UaObjectType ReferenceElementType = new UaObjectType(new NodeId(1,14),"ReferenceElementType", new LocalizedText("ReferenceElementType"), true);
    public final static UaObjectType ElementCollectionType = new UaObjectType(new NodeId(1,15),"ElementCollectionType", new LocalizedText("ElementCollectionType"), true);
    public final static UaObjectType ElementListType = new UaObjectType(new NodeId(1,16),"ElementListType", new LocalizedText("ElementListType"), true);
    public final static UaObjectType EventElementType = new UaObjectType(new NodeId(1,17),"EventElementType", new LocalizedText("EventElementType"), true);
}

File: opcua-webservice-node/source/src/server/addressspace/nodes/builtin/UaObjectTypes.ts
import { ObjectTypeIds, UaLocalizedText, UaNodeId } from "opcua-webapi-ts";
import { UaObjectType } from "../UaObjectType";

export class UaObjectTypes {
    static readonly BaseObjectType = new UaObjectType(UaNodeId.from(ObjectTypeIds.BaseObjectType), "BaseObjectType", new UaLocalizedText("BaseObjectType"), false);
    static readonly FolderType = new UaObjectType(UaNodeId.from(ObjectTypeIds.FolderType), "FolderType", new UaLocalizedText("FolderType"), false);
    static readonly ServerType = new UaObjectType(UaNodeId.from(ObjectTypeIds.ServerType), "ServerType", new UaLocalizedText("ServerType"), false);
    static readonly ModellingRuleType = new UaObjectType(UaNodeId.from(ObjectTypeIds.ModellingRuleType), "ModellingRuleType", new UaLocalizedText("ModellingRuleType"), false);
    static readonly BaseEventType = new UaObjectType(UaNodeId.from(ObjectTypeIds.BaseEventType), "BaseEventType", new UaLocalizedText("BaseEventType"), false);

    static readonly DigitalTwinType = new UaObjectType(new UaNodeId(10,1), "DigitalTwinType", new UaLocalizedText("DigitalTwinType"), true);
    static readonly DigitalTwinRepositoryType = new UaObjectType(new UaNodeId(11,1), "DigitalTwinRepositoryType", new UaLocalizedText("DigitalTwinRepositoryType"), true);
    static readonly SubmodelType = new UaObjectType(new UaNodeId(12,1), "SubmodelType", new UaLocalizedText("SubmodelType"), true);
    static readonly ElementType = new UaObjectType(new UaNodeId(13,1), "ElementType", new UaLocalizedText("ElementType"), true);
    static readonly ReferenceElementType = new UaObjectType(new UaNodeId(14,1), "ReferenceElementType", new UaLocalizedText("ReferenceElementType"), true);
    static readonly ElementCollectionType = new UaObjectType(new UaNodeId(15,1), "ElementCollectionType", new UaLocalizedText("ElementCollectionType"), true);
    static readonly ElementListType = new UaObjectType(new UaNodeId(16,1), "ElementListType", new UaLocalizedText("ElementListType"), true);
    static readonly EventElementType = new UaObjectType(new UaNodeId(17,1), "EventElementType", new UaLocalizedText("EventElementType"), true);
}